kumascript/macros/DeprecatedGeneric.ejs

```diff
@@ -23,13 +23,7 @@ if ($1) {
     if ( string.startswith(t, "js") ) {
         n = string.trim( string.substr(t, 2) );
         ver = "JavaScript ";
-        if (string.length(n) > 0) {
-            if(wiki.pageExists(lang + '/docs/JavaScript/New_in_JavaScript/' + n)) {
-                link = '/' + lang + '/docs/JavaScript/New_in_JavaScript/' + n;
-            } else if(wiki.pageExists('/en-US/docs/JavaScript/New_in_JavaScript/' + n)) {
-                link = '/en-US/docs/JavaScript/New_in_JavaScript/' + n;
-            }
-        }
+        
```

Comment:
Considering that `wiki.pageExists()` use to always return `true` no matter what, wouldn't the new code be:
```javascript
        if (string.length(n) > 0) {
            if(true) {
                link = '/' + lang + '/docs/JavaScript/New_in_JavaScript/' + n;
            } else if(true) {
                link = '/en-US/docs/JavaScript/New_in_JavaScript/' + n;
            }
        }
```
?

Reply:
Yes, that makes more sense. Rather than removing the logic and tweaking the tests. I've updated this locally.